tests/test_nanopub.py

```diff
@@ -9,194 +9,211 @@
 
 client = NanopubClient(use_test_server=True)
 
-def test_nanopub_construction_with_bnode_introduced_concept():
-    """
-    Test Nanopub construction from assertion where a BNode is introduced as a concept.
-    """
-    test_uri = 'http://www.example.com/my-nanopub'
-    test_concept_uri = 'http://www.example.com/my-nanopub#DrBob'  # This nanopub introduced DrBob
-    assertion_rdf = rdflib.Graph()
-    assertion_rdf.add((rdflib.term.BNode('DrBob'),
-                       namespaces.HYCL.claims,
-                       rdflib.Literal('This is a test')))
-
-    nanopub = Nanopub.from_assertion(
-        assertion_rdf=assertion_rdf,
-        uri=rdflib.term.URIRef(test_uri),
-        introduces_concept=rdflib.term.BNode('DrBob'),
-        derived_from=rdflib.term.URIRef('http://www.example.com/another-nanopub')
-        )
-    assert str(nanopub.introduces_concept) == test_concept_uri
+TEST_ASSERTION = (namespaces.AUTHOR.DrBob, namespaces.HYCL.claims, rdflib.Literal('This is a test'))
+TEST_ORCID = rdflib.URIRef('http://example.org/pietje')
```

Comment:
This doesn't match with what get_orcid returns right?

Reply:
Also I find the example confusing, because I would expect an ORCID iD like 0000-0000-0000-0000.

Reply:
Yeah we need to discuss what is considered the actual ORCID ID

Reply:
You are right that the orcid id is the URI + code. `get_orcid` returns a string. So I propose:
```python
TEST_ORCID_ID = 'https://orcid.org/0000-0000-0000-0000'
```